packages/playwright/src/generators/configuration/files/tsconfig.json.template

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Can we move it out of `files` folder in case the project already has a `tsconfig.json` in it? The `@nx/playwright:configuration` generator can be used to configure existing projects.